### PR TITLE
fund raiser / donations

### PR DESCRIPTION
**Pull request by RaitaroH**
@MaximeRF I don't have time at the moment to check if this breaks anything but I will leave it as a PR so I remember it. Take a look. Ex [link](https://www.youtube.com/watch?v=EPjrFjAxwlw).

### CODE REVIEW

**Comment by MaximeRF** [comment]
@RaitaroH Just tested and here are the screenshots:

When the theme is not used:
yt-donations-not-themed

The the theme is active:
yt-donations-themed

I'll check what's wrong as soon as possible.

**Comment by RaitaroH** [comment]
Wait... what? How. What is the layout you got there.

This is 50% of screen used:
image
This is full screen width:
image

Is it because being logged in changes something? That is still not it for me.

File: YouTubeDeepDarkMaterial.user.css [hunk 1]
```diff
@@ -2193,6 +2193,59 @@
...
+	#header-section.ytd-donation-shelf-renderer .style-scope.ytd-donation-shelf-renderer.no-transition
+	{
+		filter: invert(.8);
+	}
+	#donate-section.ytd-donation-shelf-renderer,
+	#creator-messages-section.ytd-donation-shelf-renderer, #nonprofit-section.ytd-donation-shelf-renderer,
+	#nonprofit-title.ytd-donation-shelf-renderer
+	{
+		background-color: var(--second-background);
+		border-color: var(--hover-background);
```
Comment (MaximeRF) [inline YouTubeDeepDarkMaterial.user.css]: The `!important` rule is missing, that's the only reason I had issues. Once added you can merge the PR.

**Comment by MaximeRF** [comment]
@RaitaroH Please check this https://github.com/RaitaroH/YouTube-DeepDark/pull/27/commits/882b65cb9f31de84708b1c7f6861eb986194da67#r332878755

File: YouTubeDeepDarkMaterial.user.css (hunk 1 above)
Comment (RaitaroH) [reply YouTubeDeepDarkMaterial.user.css]: I see. Forgot :sweat_smile:

**Pull request merged**